client/scripts/app.js

```diff
@@ -57,6 +66,13 @@ class Nametag extends Component {
       window.location = postAuth
       window.localStorage.removeItem('postAuth')
     }
+
+    // Attach user id to Sentry error reports
+    client.watchQuery({query: USER_QUERY}).subscribe({
```

Comment:
Is it alright if I move this to a Redux function? In all cases where the user is logged in we're already getting this data, and I'd like to avoid having a duplicative call to the server. Creating a function that's something like ``setSentryUser(data.me.id)`` that can be called when that data is available might be cleaner.

Happy to pair to show you where this happens if you'd like to do it.